web/main/serializers.py

```diff
@@ -57,6 +57,42 @@ def to_representation(self, value):
         return serializer.data
 
 
+class ContentNodeSerializer(serializers.ModelSerializer):
```

Comment:
True? This is identical to `SectionOutlineSerializer` except for `children = RecursiveField(many=True, allow_null=True, default=[])` -> `children = []` (and therefore no `children` in the `Meta`, and the presence of `ordinals` as well as `ordinal_string`?

If True, can `SectionOutlineSerializer` be removed entirely?

Reply:
True and True!